pyproject.toml

```diff
@@ -190,19 +165,55 @@ filename = "sdv/__init__.py"
 search = "__version__ = '{current_version}'"
 replace = "__version__ = '{new_version}'"
 
-[tool.isort]
-line_length = 99
-lines_between_types = 0
-multi_line_output = 4
-use_parentheses = true
-
-[tool.pydocstyle]
-convention = 'google'
-add-ignore = ['D105', 'D107', 'D407', 'D417']
-
 [tool.pytest.ini_options]
 addopts = "--ignore=pyproject.toml"
 
 [build-system]
 requires = ['setuptools', 'wheel']
 build-backend = 'setuptools.build_meta'
+
+[tool.ruff]
+preview = true
+line-length = 100
```

Comment:
I noticed that our other libraries with `Ruff` use this line-length, so I'm fine with increasing it. I'm just curious if there's a new industry standard. 

Fun fact: The [pep-8 standard](https://peps.python.org/pep-0008/#maximum-line-length) was 72-79. However this has been the most 'overwritten' rule.